person.rb

```diff
@@ -2,13 +2,14 @@
 
 class Person
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age:, name: 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
* Since The attribute @rentals is what holds the relationship and to manage it we will also create a method to add rentals to it.
* Note ✍️ `The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association` [Kindly read this article on many-to-many relationship](
`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship`)

Reply:
The person is being sent as a parameter to create a new rental already
```js
class Rental
  attr_accessor :date
  attr_reader :book, :person

  def initialize(date, book, person)
    @date = date

    @book = book
    book.rentals << self

    @person = person
    person.rentals << self
  end
...
```